diff --git a/accounts/users/passwd.go b/accounts/users/passwd.go
index 33139516..47daf5e7 100644
--- a/accounts/users/passwd.go
+++ b/accounts/users/passwd.go
@@ -166,7 +166,7 @@ func updatePasswd(password, username string) error {
 	}
 
 	usr, _ := user.Lookup("root")
-	grp, _ := user.LookupGroup("shadow")
+	grp, _ := user.LookupGroup("root")
 	uid, _ := strconv.Atoi(usr.Uid)
 	gid, _ := strconv.Atoi(grp.Gid)
 	err = os.Chown(userFileShadow, uid, gid)
